includes/TypesForSurfaces/SurfaceMeshRefinement.js

```diff
@@ -1,8 +1,67 @@
+if (!RFEM) {
+    throw new Error("This script is only for RFEM, it creates surface mesh refinements.");
+}
+
+/**
+* Creates surface mesh refinement
+* @class
+* @constructor
+* @param	{Number}	no				Index of surface stiffness modification, can be undefined
+* @param	{Array}		surface_list	List of surface indexes
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params  		Surface stiffness modification's parameters, can be undefined
+* @returns	Created surface mesh refinement
+*/
 function SurfaceMeshRefinement(no,
-                               comment,
-                               params)
-{
-    var surface_mesh_refinement = engine.create_surface_mesh_refinement(no);
-    set_comment_and_parameters(surface_mesh_refinement, comment, params);
-    return surface_mesh_refinement;
+	surface_list,
+	comment,
+	params) {
+	if (arguments.length !== 0) {
+		return this.surface_mesh_refinement = createSurfaceMeshRefinement(no, surface_list, comment, params);
+	}
 }
+
+/**
+* Sets target length of surface mesh refinement
+* @param	{Number}	no				Index of surface stiffness modification, can be undefined
+* @param	{Array}		surface_list	List of surface indexes
+* @param	{Number}	target_length	Target FE length
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params  		Surface stiffness modification's parameters, can be undefined
+* @returns	Created surface mesh refinement
+*/
+SurfaceMeshRefinement.prototype.TargetLength = function (no,
```

Comment:
I think we could add target lenght direcly to constructor as it is only one possible option